### ISSUE TITLE
Not working in github action

### ISSUE THREAD

**Issue by rahulbreezo** [SvanBoxel/delete-merged-branch#200]
I am trying to implement this using github action but its not working.
workflow action keeps on running with progress status neither fails nor succeeds 

My action template:
```
name: Delete Merged Branches

on: [pull_request]

jobs:
  delete:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout
        uses: actions/checkout@v2
        with:
          # Make sure the actual branch is checked out when running on pull requests
          ref: ${{ github.head_ref }}

      - name: Delete merged branch
        uses: SvanBoxel/delete-merged-branch@v1.3.3
        with:
          exclude: staging
          delete_closed_pr: false
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}

```

**Comment by SvanBoxel** [comment]
Hey @rahulbreezo I see you closed the issue. Is it working now?

**Comment by rahulbreezo** [comment]
@SvanBoxel  No it's not working.
 But now we are using bot instead of github action.

Did not expect a reply so soon, so closed the PR.

**Comment by SvanBoxel** [comment]
Fixed in https://github.com/SvanBoxel/delete-merged-branch/pull/201
